app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
   },
   NetworkController: {
-    networkId: true,
     networkStatus: true,
+    selectedNetworkClientId: false,
+    networkId: true,
     providerConfig: {
-      nickname: true,
-      ticker: true,
+      rpcUrl: false,
       type: true,
+      chainId: false,
+      ticker: true,
+      nickname: true,
+      rpcPrefs: false,
+      id: false,
+    },
+    networkConfigurations: false,
+    networksMetadata: {
+      EIPS: true,
+      status: true,
     },
   },
+  NftController: {
+    allNftContracts: false,
+    allNfts: false,
+    ignoredNfts: false,
+  },
   OnboardingController: {
     completedOnboarding: true,
     firstTimeFlowType: true,
     seedPhraseBackedUp: true,
   },
   PreferencesController: {
-    currentLocale: true,
     featureFlags: true,
-    forgottenPassword: true,
+    // double-check if contains PID?
     ipfsGateway: true,
-    preferences: true,
+    identities: false,
+    lostIdentities: false,
+    selectedAddress: false,
+    useTokenDetection: true,
+    useNftDetection: true,
+    openSeaEnabled: true,
+    isMultiAccountBalancesEnabled: true,
+    disabledRpcMethodPreferences: true,
+    showTestNetworks: true,
+    isIpfsGatewayEnabled: true,
     useBlockie: true,
     useNonceField: true,
     usePhishDetect: true,
+    dismissSeedBackUpReminder: true,
+    useMultiAccountBalanceChecker: true,
+    use4ByteResolution: true,
+    useCurrencyRateCheck: true,
+    advancedGasFee: true,
+    incomingTransactionsPreferences: true,
+    // double-check if contains PID?
+    knownMethodData: true,
```

Comment:
Maybe better to omit this. It would hint at which transactions the user had encountered

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/db9e2566e13811c8ddb6c590cc4370f8e74163f9